### DISCUSSION TITLE
Why is it so quiet here?

### DISCUSSION THREAD

**Discussion by joethephish** [inkle/ink#653] [General]
We've just migrated over from using the Issues page for general discussion so the page is looking a bit bare right now. But that doesn't mean we're not listening!

You can also find help on [our Discord server](https://discord.gg/inkle).

**Comment by vicevolf** [comment]
🤔 Because this seems to be a new feature of GitHub, many people don't realize it.

**Reply by joethephish** [reply]
Yep!

**Comment by JasonWarrenUK** [comment]
Blissful, blissful silence. Before they come. The vultures.

...

I can hear them. Hide.
